client/www/scripts/modules/manager/manager.controllers.js

```diff
@@ -352,10 +353,53 @@ Manager.controller('ManagerMainController', [
         },
         {
           id: MANAGER_CONST.LICENSE_PUSH_ACTION,
-          label: 'push license'
+          label: 'push license',
+          handler: function(host) {
```

Comment:
Oops, I'll have to fix that next time I have to make some changes to the PM manager code.